lms/static/scripts/frontend_apps/components/GradingControls.tsx

```diff
@@ -3,28 +3,29 @@ import { useCallback, useEffect, useMemo, useState } from 'preact/hooks';
 
 import { apiCall } from '../utils/api';
 import { useConfig } from '../config';
-import type { GradingConfig, StudentInfo } from '../config';
+import type { StudentInfo } from '../config';
 import { ClientRPC, useService } from '../services';
 import StudentSelector from './StudentSelector';
 import SubmitGradeForm from './SubmitGradeForm';
 
 export type GradingControlsProps = {
-  grading: GradingConfig;
+  /** List of students to grade. */
+  students: StudentInfo[];
```

Comment:
Maybe we can call this `studentsToGrade` and then we don't need the comment?

Reply:
The comment is probably superfluous anyway, given the context. eg. if you see `<GradingControls students={...}/>` it should be reasonably obvious what the list is used for. Something that is missing from the comment is any note about the sort order.